frontend/src/main/web/src/components/OverlayTrigger/index.jsx

```diff
@@ -0,0 +1,294 @@
+import React, { cloneElement, Component, PropTypes } from 'react'
+import ReactDOM from 'react-dom'
+import contains from 'dom-helpers/query/contains'
+import warning from 'warning'
+import { pick } from 'lodash'
+import createChainedFunction from '../../utils/createChainedFunction'
+import Overlay from '../Overlay'
+/**
+ * Check if value one is inside or equal to the of value
+ *
+ * @param {string} one
+ * @param {string|array} of
+ * @returns {boolean}
+ */
+function isOneOf (one, of) {
```

Comment:
The function and parameter names are just confusing. Something like this would be more readable:

```
function isOrContains (item, arrayOrValue) {
  ...
}
```

I would also refactor to swap the position of the parameters, since it reads a bit like "This array or value is or contains this item"